src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --hover-color: #00acdc;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.ul {
+  padding: 0;
+  font-size: 0;
+}
+
+.nav-item {
+  display: inline-block;
+  height: 60px;
+  padding-top: 22.5px;
```

Comment:
looks not good. 
1) You can create your nav-list as flex container with align-items:center
2) your links should be inline-block or block with top/bottom paddings